.travis.yml

```diff
@@ -16,6 +16,15 @@ services:
 install: ./service/gcs/setup_test_env
 
 script:
+  # Server-side checks
   - cd service && make
   - go test ./gcsutils/...
   - cd gcs && sudo env "PATH=$PATH" ginkgo -r -race -keepGoing
+
+  # Client-side checks
+  - cd ../../client
+  - go get -u github.com/golang/lint/golint 
```

Comment:
I recommend move those two "go get" operations into ./service/gcs/setup_test_env, which has all the test prep stuffs.